x-pack/legacy/plugins/apm/public/components/app/ErrorGroupOverview/List/index.tsx

```diff
@@ -59,9 +59,26 @@ const ErrorGroupList: React.FC<Props> = props => {
   const columns = useMemo(
     () => [
       {
-        name: i18n.translate('xpack.apm.errorsTable.groupIdColumnLabel', {
-          defaultMessage: 'Group ID'
-        }),
+        name: (
+          <>
+            {i18n.translate('xpack.apm.errorsTable.groupIdColumnLabel', {
+              defaultMessage: 'Group ID'
+            })}{' '}
+            <EuiIconTip
+              size="m"
```

Comment:
```suggestion
              size="s"
```
Aligning with Transactions list table (Impact tooltip)